taskcluster/ac_taskgraph/transforms/beetmover.py

```diff
@@ -31,7 +32,24 @@ def resolve_keys(config, tasks):
 
 @transforms.add
 def set_artifact_map(config, tasks):
+
+    def _craft_path_version(version, build_type, nightly_version):
+        """Helper function to craft the correct version to bake in the artifacts full
+        path section"""
+        ret = "{}{}".format(
+            version,
+            "-SNAPSHOT" if build_type == "snapshot" else ''
+        )
+        # XXX: for nightly releases we need to s/X.0.0/X.0.<buildid>/g for the
+        # version within the destination path
+        # e.g. maven2/org/mozilla/components/browser-awesomebar/34.0.20200212190116/
+        if build_type == 'nightly':
+            if version in ret:
```

Comment:
Out of curiosity, what are the cases where `version` is not in `ret` but the build is still a nightly?

Nit: If we keep this condition, we can combine with the line above.

Reply:
Good catch. We always have the version set there so that `if` is useless.